src/node.rs

```diff
@@ -1012,15 +1084,16 @@ mod tests {
         );
         assert_eq!(state1.seq_num, 0);
 
-        // Try a block way in the future (push to backlog)
-        let mut node1 = mock_node(1);
-        let mut state1 = PbftState::new(1, &cfg);
-        node1
-            .on_block_new(mock_block(7), &mut state1)
-            .unwrap_or_else(handle_pbft_err);
-        assert_eq!(state1.phase, PbftPhase::NotStarted);
-        assert_eq!(state1.working_block, WorkingBlockOption::NoWorkingBlock);
-        assert_eq!(state1.seq_num, 0);
+        // TODO: Currently broken
```

Comment:
@aludvik: Since we can't/shouldn't create the `Consensus*` protobuf messages ourselves, we can't test this without having logic in the test function specific to unit tests, which isn't great. On the other hand, if we allowed PBFT to create `Consensus*` protobuf messages, that would allow testing here, as well as fixing the issue elsewhere where a node will send itself a message and we currently just insert a special `"myself"` value to indicate where the message came from.